build-images.sh

```diff
@@ -1,11 +1,8 @@
 #!/usr/bin/env bash
 
-docker build --file 7.3/Dockerfile -t cimg/php:7.3.32  -t cimg/php:7.3 .
-docker build --file 7.3/node/Dockerfile -t cimg/php:7.3.32-node  -t cimg/php:7.3-node .
-docker build --file 7.3/browsers/Dockerfile -t cimg/php:7.3.32-browsers  -t cimg/php:7.3-browsers .
-docker build --file 7.4/Dockerfile -t cimg/php:7.4.25  -t cimg/php:7.4 .
-docker build --file 7.4/node/Dockerfile -t cimg/php:7.4.25-node  -t cimg/php:7.4-node .
-docker build --file 7.4/browsers/Dockerfile -t cimg/php:7.4.25-browsers  -t cimg/php:7.4-browsers .
-docker build --file 8.0/Dockerfile -t cimg/php:8.0.13  -t cimg/php:8.0 .
-docker build --file 8.0/node/Dockerfile -t cimg/php:8.0.13-node  -t cimg/php:8.0-node .
-docker build --file 8.0/browsers/Dockerfile -t cimg/php:8.0.13-browsers  -t cimg/php:8.0-browsers .
```

Comment:
Did this cause these images to stop being available?

Reply:
No. That just means those tags won't be built (and possible published) with this PR is merged.

Running into an issue? If so, the new tags we're publishing have some changes to how we get PHP which can cause some changes. There's one Issue opened already: https://github.com/CircleCI-Public/cimg-php/issues/130

Feel free to add on to that or open a new Issue if needed.